src/Pact/Types/ChainMeta.hs

```diff
@@ -75,16 +77,30 @@ instance FromJSON PrivateMeta where parseJSON = lensyParseJSON 3
 instance NFData PrivateMeta
 instance Serialize PrivateMeta
 
+
+-- | Contains ChainId, Chainweb network version, ect.
+--   Meant to prevent sending Testnet txs to Mainet.
```

Comment:
Change comment to "Expresses unique platform-specific chain identifier" as this will differ on different platforms.